test/executor.test.js

```diff
@@ -468,9 +468,27 @@ describe('executor', function() {
 
     boot(app, appdir.PATH);
 
+    expect(Object.keys(require.cache)).to
+    .include(appdir.resolve('components/test-component/index.js'));
+
     expect(app.componentOptions).to.eql({ option: 'value' });
   });
 
+  it('disables component when configuration is not set', function() {
+    appdir.writeConfigFileSync('component-config.json', {
+      './components/test-component1': false
```

Comment:
Now that you are using `appdir.resolve`, is it still needed to use a unique name instead of `./components/test-component`?